assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -267,6 +267,9 @@ che.infra.docker.bootstrapper.server_check_period_sec=3
 # in parallel on workspace startups.
 che.infra.docker.max_pull_threads=10
 
+# Time(in seconds) that limits the docker build process.
+# The default value is 8 minutes, after which the build will be considered as failed.
+che.infra.docker.build_timeout_sec=480
```

Comment:
How much time does it take to build dockerfile of Che master in some good case?

Reply:
in master, we do not have any stacks that contain workspace config with dockerfile recipe type but form my custom workspace config docker image build does not take more than 3 minutes. The build process depends on many factors so it is hard to estimate the default value, any suggestions are welcome.